one_collect/src/helpers/callstack/os/linux.rs

```diff
@@ -234,21 +241,25 @@ impl<'a> UnwindRequest<'a> {
 
     pub fn unwind_machine(
         &mut self) -> UnwindResult {
-        self.machine.unwind_process(
+        trace!("Starting machine unwind: pid={}, rip={:#x}, rbp={:#x}, rsp={:#x}", self.pid, self.rip, self.rbp, self.rsp);
+        let result = self.machine.unwind_process(
```

Comment:
These trace! statements are run per unwind operation, which will happen very often. I think we can safely remove the per-unwind operations except for the ones that I'll flag below.